lib/omni_exchange/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module OmniExchange
-  VERSION = '2.0.0'
+  VERSION = '3.0.0'
```

Comment:
Since we replaced the used standard library used in Ruby versions below `3.0.0` with a gem I believe this might be a breaking change...  but not absolutely sure here so comments here are appreciated.

Reply:
Bumping major version sounds good, the new release doesn't support Ruby 2.7.